Makefile

```diff
@@ -279,10 +281,23 @@ ${SNITCH_INSTALL_DIR}: ${TOOLCHAIN_DIR}/snitch_cluster
 	TMPDIR=tmp pip install -r python-requirements.txt && rm -rf tmp && \
 	bender vendor init && \
 	cd ${SNITCH_INSTALL_DIR}/target/snitch_cluster && \
-	make sw/runtime/banshee sw/math
+	make sw/runtime/banshee sw/runtime/rtl sw/math
 
 snitch_runtime: ${SNITCH_INSTALL_DIR}
 
+${TOOLCHAIN_DIR}/gvsoc:
+	cd ${TOOLCHAIN_DIR} && \
+	git clone https://github.com/gvsoc/gvsoc.git && \
+	cd ${TOOLCHAIN_DIR}/gvsoc &&  \
+	git submodule update --init --recursive && \
+	pip install -r core/requirements.txt && pip install -r gapy/requirements.txt
```

Comment:
Please add a specific commit hash that you have tested against. We want to have full control over the environment. Without it, any change to the master could break our tests.